iocore/net/SSLUtils.cc

```diff
@@ -2007,7 +2007,10 @@ SSLParseCertificateConfiguration(const SSLConfigParams *params, SSLCertLookup *l
         if (ssl_extract_certificate(&line_info, sslMultiCertSettings)) {
           // There must be a certificate specified unless the tunnel action is set
           if (sslMultiCertSettings.cert || sslMultiCertSettings.opt != SSLCertContext::OPT_TUNNEL) {
-            ssl_store_ssl_context(params, lookup, &sslMultiCertSettings);
+            if (ssl_store_ssl_context(params, lookup, &sslMultiCertSettings) == nullptr) {
+              Error("failed to load SSL server contexts");
+              return false;
+            }
           } else {
```

Comment:
I'd be fine adding additional actionable checks here if you like, I'm just not familiar with most of this code.